src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,52 @@
+// @flow
+import React, { useState, useCallback } from "react";
+import * as _ from "lodash";
+
+type Props = {
+  button: string,
+  label: string,
+  items: Array<string>
+};
+
+export default function Dropdown(props: Props) {
+  const { button, label, items } = props;
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [selectedItem, setSelectedItem] = useState(button);
+  const expandMenu = useCallback(() => {
+    setIsExpanded(s => !s);
+  }, [setIsExpanded]);
+  const selectItem = useCallback(
+    item => {
+      setIsExpanded(s => !s);
+      setSelectedItem(item.value);
+    },
+    [setIsExpanded, setSelectedItem]
+  );
+  return (
```

Comment:
Add blank line - you need to separate blocks of code a bit more - code should be readable.